datafusion/src/logical_plan/plan.rs

```diff
@@ -263,19 +292,9 @@ pub enum LogicalPlan {
     /// Produces rows from a table provider by reference or from the context
     TableScan(TableScanPlan),
     /// Produces no rows: An empty relation with an empty schema
-    EmptyRelation {
-        /// Whether to produce a placeholder row
-        produce_one_row: bool,
-        /// The schema description of the output
-        schema: DFSchemaRef,
-    },
+    EmptyRelation(EmptyRelation),
     /// Produces the first `n` tuples from its input and discards the rest.
-    Limit {
-        /// The limit
-        n: usize,
-        /// The logical plan
-        input: Arc<LogicalPlan>,
-    },
+    Limit(Limit),
```

Comment:
Using a different name will be better, such as `LimitPlan`, keep the same as others.

Reply:
I actually think the way this PR is written, `Limit(Limit)` is more consistent.  The only one that seems to follow a  different pattern is `TableScanPlan`  (which we should perhaps rename to `TableScan`?)